docs/Performance.md

```diff
@@ -0,0 +1,242 @@
+# Protobuf Perforamcne
+This benchmark result is tested on workstation with processor of Intel® Xeon® Processor E5-2630 and 32GB RAM
+
+## Parsing performance
+<table>
+<tbody><tr>
+<th rowspan="2"> </th>
+<th colspan="3" rowspan="1">cpp</th>
+<th colspan="3" rowspan="1">java</th>
+<th colspan="3" rowspan="1">python</th>
+</tr>
+<tr>
+<th colspan="1">new(use new operator for creating message instance)</th>
+<th colspan="1">newarena(using arena for creating new message instance)</th>
+<th colspan="1">reuse(reuse the same message instance for parsing)</th>
+<th colspan="1">deserializeFromByteArray</th>
```

Comment:
use simpler names for java? For example, byte[], ByteString, InputStream.

Reply:
Done